commands/apps/claude/claude.js

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+// Dependency: This script requires `chrome-cli` installed: https://github.com/prasmussen/chrome-cli
+// Install via homebrew: `brew install chrome-cli`
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Prompt Claude
+// @raycast.mode silent
+// @raycast.packageName Claude
+
+// Optional parameters:
+// @raycast.icon 🧠
+// @raycast.argument1 { "type": "text", "placeholder": "prompt"}
```

Comment:
The placeholders need to be camel case

```suggestion
// @raycast.argument1 { "type": "text", "placeholder": "Prompt"}
```